feature-libs/product-configurator/rulebased/cpq/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from './cpq-configurator-normalizer';
+export * from './cpq-configurator-overview-normalizer';
+export * from './cpq-configurator-rest.adapter';
+export * from './cpq-configurator-rest.module';
+export * from './cpq-configurator-rest.service';
+export * from './cpq-configurator-serializer';
+export * from './cpq-configurator-utils.service';
+export * from './cpq-configurator-value-serializer';
+export * from './cpq-configurator.converters';
+export * from './cpq.models';
```

Comment:
This folder is quite busy with items of 2 types (converters and rest adapters/services). For clarity Can we create a subfolder `/converters`, please? It could contain normalizers, serialziers, as well as the [normalizer utils (see another comment)](https://github.com/SAP/spartacus/pull/11001#discussion_r609670378)

Reply:
Picked!

Reply:
Done!